packages/loader/container-loader/src/container.ts

```diff
@@ -1027,6 +1035,7 @@ export class Container
 			this.storageAdapter,
 			offlineLoadEnabled,
 			this,
+			() => this.isStorageOnly,
```

Comment:
let's just inline the function here, rather than add single use private getter to the class.